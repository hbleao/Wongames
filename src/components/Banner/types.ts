export type ImageProps = {
  src: string;
};

export type BannerProps = {
  image: string;
  title: string;
  subtitle: string;
  buttonLabel: string;
  buttonLink: string;
};
