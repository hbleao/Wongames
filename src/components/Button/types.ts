import React, {
  AnchorHTMLAttributes,
  ButtonHTMLAttributes,
  ReactNode
} from 'react';

type Size = 'small' | 'medium' | 'large';
type ButtonTypes =
  | AnchorHTMLAttributes<HTMLAnchorElement>
  | ButtonHTMLAttributes<HTMLButtonElement>;

export type ButtonProps = {
  size?: Size;
  fullWidth?: boolean;
  icon?: ReactNode;
  as?: React.ElementType;
} & ButtonTypes;

export type WrapperProps = {
  fullWidth?: boolean;
  size?: Size;
  hasIcon: boolean;
};
