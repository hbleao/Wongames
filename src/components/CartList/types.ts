import { GameItemProps } from 'components/GameItem/types';

export type CartListProps = {
  items: GameItemProps[];
  total: string;
};
