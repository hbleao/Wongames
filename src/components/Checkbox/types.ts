import { InputHTMLAttributes } from 'react';

export type CheckboxProps = {
  labelText?: string;
  labelFor?: string;
  isChecked?: boolean;
  labelColor?: 'black' | 'white';
  value?: string | ReadonlyArray<string> | number;
  onCheck?: (status: boolean) => void;
} & InputHTMLAttributes<HTMLInputElement>;

export type LabelProps = {
  labelColor?: 'black' | 'white';
};
