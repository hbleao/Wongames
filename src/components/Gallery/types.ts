export type GalleryImageProps = {
  src: string;
  alt: string;
  label: string;
};

export type GalleryProps = {
  items: GalleryImageProps[];
};

export type ModalProps = {
  isOpen: boolean;
};
