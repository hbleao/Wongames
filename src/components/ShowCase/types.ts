import { GameCardProps } from 'components/GameCard/types';
import { HighlightProps } from 'components/Highlight/types';

export type ShowCaseProps = {
  title?: string;
  titleColor?: 'white' | 'black';
  highlight?: HighlightProps;
  games?: GameCardProps[];
};
