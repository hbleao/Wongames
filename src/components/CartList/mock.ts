export const mockCartList = [
  {
    image: 'https://source.unsplash.com/user/williamjusten/151x70',
    title: 'Red Dead Redemption 2',
    price: 'R$ 215,00'
  },
  {
    image: 'https://source.unsplash.com/user/williamjusten/151x70',
    title: 'Borderlands 3',
    price: 'R$ 129,90'
  }
];
