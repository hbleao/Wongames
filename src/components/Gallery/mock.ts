export const mockGallery = [
  {
    src: '/img/games/cyberpunk-1.jpg',
    label: 'Gallery Image 1',
    alt: 'alt image 1'
  },
  {
    src: '/img/games/cyberpunk-2.jpg',
    label: 'Gallery Image 2',
    alt: 'alt image 2'
  },
  {
    src: '/img/games/cyberpunk-3.jpg',
    label: 'Gallery Image 3',
    alt: 'alt image 3'
  },
  {
    src: '/img/games/cyberpunk-4.jpg',
    label: 'Gallery Image 4',
    alt: 'alt image 4'
  },
  {
    src: '/img/games/cyberpunk-5.jpg',
    label: 'Gallery Image 5',
    alt: 'alt image 5'
  },
  {
    src: '/img/games/cyberpunk-6.jpg',
    label: 'Gallery Image 6',
    alt: 'alt image 6'
  }
];
