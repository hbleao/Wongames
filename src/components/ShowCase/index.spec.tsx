import React from 'react';
import { screen } from '@testing-library/react';

import { ShowCase } from '.';

import { renderWithTheme } from 'utils/tests/renderWithTheme';

import { ShowCaseProps } from './types';
import { mockGameCards } from 'components/GameCardSlider/mock';
import { mockHighlight } from 'components/Highlight/mock';

const makeSut = ({ title, games, highlight }: ShowCaseProps) => {
  const sut = renderWithTheme(
    <ShowCase title={title} games={games} highlight={highlight} />
  );

  return {
    sut
  };
};

describe('Components/ShowCase', () => {
  it('should be render a ShowCase', () => {
    const title = 'Top Five';
    const games = mockGameCards;
    const { sut } = makeSut({ title, games, highlight: mockHighlight });

    const titlePage = screen.getByRole('heading', { name: title });
    const highlightTitle = screen.getByRole('heading', {
      name: mockHighlight.title
    });
    const gameCardTitle = screen.getByRole('heading', {
      name: mockGameCards[0].title
    });

    expect(titlePage).toBeInTheDocument();
    expect(highlightTitle).toBeInTheDocument();
    expect(gameCardTitle).toBeInTheDocument();
    expect(sut.container).toMatchSnapshot();
  });

  it('should be render a ShowCase without title', () => {
    makeSut({});

    const titlePage = screen.queryByRole('heading');

    expect(titlePage).not.toBeInTheDocument();
  });

  it('should be render a ShowCase without game card title', () => {
    makeSut({});

    const gameCardTitle = screen.queryByRole('heading', {
      name: mockGameCards[0].title
    });

    expect(gameCardTitle).not.toBeInTheDocument();
  });

  it('should be render a ShowCase without highlight title', () => {
    makeSut({});

    const highlightTitle = screen.queryByRole('heading', {
      name: mockGameCards[0].title
    });

    expect(highlightTitle).not.toBeInTheDocument();
  });
});
