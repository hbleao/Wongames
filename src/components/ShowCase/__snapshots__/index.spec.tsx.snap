// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/ShowCase should be render a ShowCase 1`] = `
.c10 {
  color: #FAFAFA;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  cursor: pointer;
  border: none;
  border-radius: 0.4rem;
  padding: 0.8rem;
  background-image: linear-gradient(180deg,#ff5f5f 0%,#f062c0 50%);
  -webkit-text-decoration: none;
  text-decoration: none;
  height: 4rem;
  font-size: 1.4rem;
  padding: 0.8rem 3.2rem;
}

.c10:hover {
  background: linear-gradient(180deg,#e35565 0%,#d958a6 50%);
}

.c25 {
  color: #FAFAFA;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  cursor: pointer;
  border: none;
  border-radius: 0.4rem;
  padding: 0.8rem;
  background-image: linear-gradient(180deg,#ff5f5f 0%,#f062c0 50%);
  -webkit-text-decoration: none;
  text-decoration: none;
  height: 3rem;
  font-size: 1.2rem;
}

.c25:hover {
  background: linear-gradient(180deg,#e35565 0%,#d958a6 50%);
}

.c25 > svg {
  width: 1.5rem;
}

.c25 > svg + span {
  margin-left: 0.8rem;
}

.c3 {
  color: #FAFAFA;
  padding-left: 0.8rem;
  border-left: 0.7rem solid #F231A5;
  font-size: 2.0rem;
}

.c13 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c15 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  height: 100%;
  background-color: #FAFAFA;
}

.c16 {
  height: 14rem;
  width: 100%;
  background: #f6f7f8;
  background-image: linear-gradient( to right,#f6f7f8 0%,#ededed 20%,#f6f7f8 40%,#f6f7f8 100% );
  background-size: 80rem 14rem;
  -webkit-animation: placeholderShimmer 1s linear infinite forwards;
  animation: placeholderShimmer 1s linear infinite forwards;
}

.c16 img {
  width: 100%;
  height: 100%;
  object-fit: cover;
}

.c17 {
  position: relative;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 1.6rem;
}

.c18 {
  max-width: calc(100% - 2.5rem);
}

.c19 {
  font-size: 1.6rem;
  line-height: 1.6rem;
  font-weight: 600;
  color: #030517;
}

.c20 {
  font-size: 1.4rem;
  font-weight: 600;
  color: #8F8F8F;
}

.c21 {
  color: #F231A5;
  position: absolute;
  right: 0;
  top: -0.5rem;
  cursor: pointer;
  background-color: transparent;
  border: none;
}

.c21 svg {
  width: 2.5rem;
}

.c22 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  margin-top: 2.4rem;
}

.c23 {
  height: 3rem;
  font-weight: 600;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #8F8F8F;
  -webkit-text-decoration: line-through;
  text-decoration: line-through;
  margin-right: 1.6rem;
}

.c24 {
  height: 3rem;
  font-weight: 600;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #FAFAFA;
  background-color: #3CD3C1;
  padding: 0 0.8rem;
  margin-right: calc(0.8rem / 2);
  border-radius: 0.4rem;
}

.c5 {
  position: relative;
  height: 23rem;
  background-image: url(/img/red-dead.png);
  background-position: center center;
  background-size: cover;
  display: grid;
  grid-template-areas: 'floatImage content';
  grid-template-columns: 1fr 2fr;
}

.c5::after {
  content: '';
  position: absolute;
  width: 100%;
  height: 100%;
  background-color: rgba(0,0,0,0.6);
}

.c5 .c6 {
  text-align: right;
}

.c7 {
  z-index: 10;
  padding: 1.6rem;
  grid-area: content;
}

.c8 {
  font-size: 1.8rem;
  font-weight: 600;
  color: #FAFAFA;
}

.c9 {
  font-size: 1.4rem;
  font-weight: 300;
  color: #FAFAFA;
  margin-bottom: 2.4rem;
}

.c12 .slick-slider {
  position: relative;
  display: block;
  box-sizing: border-box;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-touch-callout: none;
  -khtml-user-select: none;
  -ms-touch-action: pan-y;
  touch-action: pan-y;
  -webkit-tap-highlight-color: transparent;
}

.c12 .slick-list {
  position: relative;
  display: block;
  overflow: hidden;
  margin: 0;
  padding: 0;
}

.c12 .slick-list:focus {
  outline: none;
}

.c12 .slick-list.dragging {
  cursor: pointer;
  cursor: hand;
}

.c12 .slick-slider .slick-track,
.c12 .slick-slider .slick-list {
  -webkit-transform: translate3d(0,0,0);
  -moz-transform: translate3d(0,0,0);
  -ms-transform: translate3d(0,0,0);
  -o-transform: translate3d(0,0,0);
  -webkit-transform: translate3d(0,0,0);
  -ms-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}

.c12 .slick-track {
  position: relative;
  top: 0;
  left: 0;
  display: block;
}

.c12 .slick-track:before,
.c12 .slick-track:after {
  display: table;
  content: '';
}

.c12 .slick-track:after {
  clear: both;
}

.c12 .slick-loading .slick-track {
  visibility: hidden;
}

.c12 .slick-slide {
  display: none;
  float: left;
  height: 100%;
  min-height: 1px;
}

.c12 [dir='rtl'] .slick-slide {
  float: right;
}

.c12 .slick-slide img {
  display: block;
}

.c12 .slick-slide.slick-loading img {
  display: none;
}

.c12 .slick-slide.dragging img {
  pointer-events: none;
}

.c12 .slick-initialized .slick-slide {
  display: block;
}

.c12 .slick-loading .slick-slide {
  visibility: hidden;
}

.c12 .slick-vertical .slick-slide {
  display: block;
  height: auto;
  border: 1px solid transparent;
}

.c12 .slick-arrow.slick-hidden {
  display: none;
}

.c11 .slick-track,
.c11 .slick-list {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c11 .slick-slide > div {
  margin: 0 0.8rem;
  -webkit-flex: 1 0 auto;
  -ms-flex: 1 0 auto;
  flex: 1 0 auto;
  height: 100%;
}

.c11 .slick-list {
  margin: 0 -0.8rem;
}

.c11 .slick-prev,
.c11 .slick-next {
  display: block;
  color: #030517;
  cursor: pointer;
  position: absolute;
  top: 50%;
  width: 2.5rem;
  height: 2.5rem;
  padding: 0;
  -webkit-transform: translate(0,-50%);
  -ms-transform: translate(0,-50%);
  transform: translate(0,-50%);
}

.c11 .slick-prev {
  left: -5.6rem;
}

.c11 .slick-next {
  right: -5.6rem;
}

.c11 .slick-prev.slick-disabled,
.c11 .slick-next.slick-disabled {
  visibility: hidden;
}

.c0 {
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

.c1 {
  margin-bottom: calc(3.2rem * 2);
}

.c1 .c2,
.c1 .c4,
.c1 .c14 {
  margin-bottom: 3.2rem;
}

@media (min-width:768px) {
  .c3 {
    font-size: 2.8rem;
  }
}

@media (min-width:768px) {
  .c5 {
    height: 32rem;
  }
}

@media (min-width:768px) {
  .c7 {
    -webkit-align-self: end;
    -ms-flex-item-align: end;
    align-self: end;
    padding: 4.0rem;
  }
}

@media (min-width:768px) {
  .c8 {
    font-size: 2.8rem;
  }
}

@media (min-width:768px) {
  .c9 {
    font-size: 1.8rem;
  }
}

@media (max-width:1440px) {
  .c11 {
    overflow-x: hidden;
  }
}

@media (min-width:1170px) {
  .c11 .slick-slide > div {
    margin: 0 1.6rem;
  }

  .c11 .slick-list {
    margin: 0 -1.6rem;
  }
}

@media (max-width:768px) {
  .c1 .c4 {
    margin-right: calc(-3.2rem / 2);
    margin-left: calc(-3.2rem / 2);
  }
}

@media (max-width:1440px) {
  .c1 .c14 {
    margin-right: calc(-3.2rem / 2);
  }
}

<div>
  <section
    class="c0 c1"
  >
    <h2
      class="c2 c3"
      color="white"
    >
      Top Five
    </h2>
    <div
      class="c4 c5"
    >
      <div
        class="c6 c7"
      >
        <h1
          class="c8"
        >
          Read Dead is back!
        </h1>
        <h2
          class="c9"
        >
          Come see Jhon's new adventures
        </h2>
        <a
          class="c10"
          href="/rdr2"
        >
          <span>
            Buy Now
          </span>
        </a>
      </div>
    </div>
    <div
      class="c11"
      color="black"
    >
      <section
        class="c12"
      >
        <div
          class="slick-slider slick-initialized"
          dir="ltr"
        >
          <svg
            aria-hidden="true"
            aria-label="previous games"
            class="c13 slick-arrow slick-prev slick-disabled"
            data-role="none"
            fill="currentColor"
            focusable="false"
            height="20"
            style="display: block;"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
              opacity="0.87"
            />
            <path
              d="M17.51 3.87L15.73 2.1 5.84 12l9.9 9.9 1.77-1.77L9.38 12l8.13-8.13z"
            />
          </svg>
          <div
            class="slick-list"
          >
            <div
              class="slick-track"
              style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-hidden="false"
                class="slick-slide slick-active slick-current"
                data-index="0"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="c14 c15"
                  >
                    <div
                      class="c16"
                    >
                      <img
                        alt="Population 1"
                        src="https://source.unsplash.com/user/willianjusten/300x140"
                      />
                    </div>
                    <div
                      class="c17"
                    >
                      <div
                        class="c18"
                      >
                        <h3
                          class="c19"
                        >
                          Population 1
                        </h3>
                        <h4
                          class="c20"
                        >
                          Rockstar Games
                        </h4>
                      </div>
                      <button
                        aria-labelledby="Wishlist button group"
                        class="c21"
                      >
                        <svg
                          aria-hidden="true"
                          aria-label="Add to wishlist"
                          class="c13"
                          fill="currentColor"
                          focusable="false"
                          height="20"
                          viewBox="0 0 24 24"
                          width="20"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M0 0h24v24H0V0z"
                            fill="none"
                          />
                          <path
                            d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                          />
                        </svg>
                      </button>
                      <div
                        class="c22"
                      >
                        <div
                          class="c23"
                        >
                          R$ 235,00
                        </div>
                        <div
                          class="c24"
                        >
                          R$ 215,00
                        </div>
                        <button
                          aria-label="Go to shopping cart"
                          class="c25"
                        >
                          <svg
                            aria-hidden="true"
                            class="c13"
                            fill="currentColor"
                            focusable="false"
                            viewBox="0 0 24 24"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-8.9-5h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4l-3.87 7H8.53L4.27 2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2z"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="1"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="c14 c15"
                  >
                    <div
                      class="c16"
                    >
                      <img
                        alt="Population 2"
                        src="https://source.unsplash.com/user/willianjusten/300x140"
                      />
                    </div>
                    <div
                      class="c17"
                    >
                      <div
                        class="c18"
                      >
                        <h3
                          class="c19"
                        >
                          Population 2
                        </h3>
                        <h4
                          class="c20"
                        >
                          Rockstar Games
                        </h4>
                      </div>
                      <button
                        aria-labelledby="Wishlist button group"
                        class="c21"
                      >
                        <svg
                          aria-hidden="true"
                          aria-label="Add to wishlist"
                          class="c13"
                          fill="currentColor"
                          focusable="false"
                          height="20"
                          viewBox="0 0 24 24"
                          width="20"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M0 0h24v24H0V0z"
                            fill="none"
                          />
                          <path
                            d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                          />
                        </svg>
                      </button>
                      <div
                        class="c22"
                      >
                        <div
                          class="c23"
                        >
                          R$ 235,00
                        </div>
                        <div
                          class="c24"
                        >
                          R$ 215,00
                        </div>
                        <button
                          aria-label="Go to shopping cart"
                          class="c25"
                        >
                          <svg
                            aria-hidden="true"
                            class="c13"
                            fill="currentColor"
                            focusable="false"
                            viewBox="0 0 24 24"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-8.9-5h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4l-3.87 7H8.53L4.27 2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2z"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="2"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="c14 c15"
                  >
                    <div
                      class="c16"
                    >
                      <img
                        alt="Population 3"
                        src="https://source.unsplash.com/user/willianjusten/300x140"
                      />
                    </div>
                    <div
                      class="c17"
                    >
                      <div
                        class="c18"
                      >
                        <h3
                          class="c19"
                        >
                          Population 3
                        </h3>
                        <h4
                          class="c20"
                        >
                          Rockstar Games
                        </h4>
                      </div>
                      <button
                        aria-labelledby="Wishlist button group"
                        class="c21"
                      >
                        <svg
                          aria-hidden="true"
                          aria-label="Add to wishlist"
                          class="c13"
                          fill="currentColor"
                          focusable="false"
                          height="20"
                          viewBox="0 0 24 24"
                          width="20"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M0 0h24v24H0V0z"
                            fill="none"
                          />
                          <path
                            d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                          />
                        </svg>
                      </button>
                      <div
                        class="c22"
                      >
                        <div
                          class="c23"
                        >
                          R$ 235,00
                        </div>
                        <div
                          class="c24"
                        >
                          R$ 215,00
                        </div>
                        <button
                          aria-label="Go to shopping cart"
                          class="c25"
                        >
                          <svg
                            aria-hidden="true"
                            class="c13"
                            fill="currentColor"
                            focusable="false"
                            viewBox="0 0 24 24"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-8.9-5h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4l-3.87 7H8.53L4.27 2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2z"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="3"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="c14 c15"
                  >
                    <div
                      class="c16"
                    >
                      <img
                        alt="Population 4"
                        src="https://source.unsplash.com/user/willianjusten/300x140"
                      />
                    </div>
                    <div
                      class="c17"
                    >
                      <div
                        class="c18"
                      >
                        <h3
                          class="c19"
                        >
                          Population 4
                        </h3>
                        <h4
                          class="c20"
                        >
                          Rockstar Games
                        </h4>
                      </div>
                      <button
                        aria-labelledby="Wishlist button group"
                        class="c21"
                      >
                        <svg
                          aria-hidden="true"
                          aria-label="Add to wishlist"
                          class="c13"
                          fill="currentColor"
                          focusable="false"
                          height="20"
                          viewBox="0 0 24 24"
                          width="20"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M0 0h24v24H0V0z"
                            fill="none"
                          />
                          <path
                            d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"
                          />
                        </svg>
                      </button>
                      <div
                        class="c22"
                      >
                        <div
                          class="c23"
                        >
                          R$ 235,00
                        </div>
                        <div
                          class="c24"
                        >
                          R$ 215,00
                        </div>
                        <button
                          aria-label="Go to shopping cart"
                          class="c25"
                        >
                          <svg
                            aria-hidden="true"
                            class="c13"
                            fill="currentColor"
                            focusable="false"
                            viewBox="0 0 24 24"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-8.9-5h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4l-3.87 7H8.53L4.27 2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2z"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="4"
                style="outline: none; width: 0px;"
                tabindex="-1"
              />
            </div>
          </div>
          <svg
            aria-hidden="true"
            aria-label="next games"
            class="c13 slick-arrow slick-next"
            data-role="none"
            fill="currentColor"
            focusable="false"
            height="20"
            style="display: block;"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M24 24H0V0h24v24z"
              fill="none"
              opacity="0.87"
            />
            <path
              d="M6.49 20.13l1.77 1.77 9.9-9.9-9.9-9.9-1.77 1.77L14.62 12l-8.13 8.13z"
            />
          </svg>
        </div>
      </section>
    </div>
  </section>
</div>
`;
